packages/rolldown/src/binding.d.ts

```diff
@@ -606,6 +628,25 @@ export interface IsolatedDeclarationsOptions {
   sourcemap?: boolean
 }
 
+export interface IsolatedDeclarationsOptions {
```

Comment:
Related to #3058 

I tried what you did on windows, but it didn't work. @hi-ogawa cc